pkg/latte/intent_login.go

```diff
@@ -85,7 +86,15 @@ func (i *IntentLogin) ReactTo(ctx context.Context, deps *workflow.Dependencies,
 					authenticator.KeepKind(authenticator.KindPrimary),
 					authenticator.KeepType(model.AuthenticatorTypePassword),
 				)
-				if err != nil {
+				if errors.Is(err, api.ErrNoAuthenticator) {
+					// allow the user to start the intent
+					// if the user doesn't have a password authenticator
+					// the node will always return invalid credentials error
+					// when receiving the password
+					return workflow.NewSubWorkflow(&IntentAuthenticatePassword{
+						Authenticator: nil,
```

Comment:
The workflow is stuck for the following case:

1. Create the user without password
2. Start the sign-in workflow
3. The user resets (or we should set up) the password
4. The intent and the node has stale information